packages/@ember/-internals/glimmer/lib/helpers/component.ts

```diff
@@ -106,6 +110,12 @@
   When yielding the component via the `hash` helper, the component is invoked directly.
   See the following snippet:
 
+  ```
+  <PersonForm as |form|>
+    {{form.nameInput placeholder="Username"}}
```

Comment:
```suggestion
    <form.nameInput @placeholder={{"Username"}} />
```

Reply:
```suggestion
    <form.nameInput @placeholder="Username" />
```

Reply:
@wycats which of these is preferable?

Reply:
I used <form.nameInput @placeholder={{"Username"}} /> for now

Reply:
Generally speaking there is no need for curlies around a static string. Its not "wrong", just a bit odd...